styles.css

```diff
@@ -1 +1,77 @@
-/* this will be the styles file */
+* {
+  background-color: black;
+}
+body {
+  margin: 0px 160px;
+}
+.logo {
+  display: flex;
+  color: #fff;
+  justify-self: start;
+  align-items: center;
+}
+nav {
+  color: #fff;
+  display: flex;
+  text-align: center;
```

Comment:
Is this required?
can we use justify instead?

Reply:
done